app/v2/api_spec/get_api_spec.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from flask import Blueprint, current_app, send_file
+
+v2_api_spec_blueprint = Blueprint("v2_api_spec", __name__, url_prefix="/v2")
+
+
+@v2_api_spec_blueprint.route("openapi-en", methods=["GET"])
```

Comment:
Missing the leading `/` here.  It still seems to work fine, but let's make it consistent.